Tidies up language & consolidates guidelines. (#1018)
TheOdinProject/theodinproject: doc/code_of_conduct.md

The Odin Project's online communities are intended to be informative and relaxing spaces where Odinites can find answers to their programming questions or simply hang out.  Since we aim to be an inclusive and friendly community, we request that all Odinites follow the guidelines laid out in this Code of Conduct.  Following these guidelines will ensure that all Odinites have a positive learning experience:

> 1. Be polite, welcoming, and inclusive.  Inappropriate conduct or harassment will not be tolerated. Racisism, sexual harrasment, and religious persecution will not be tolerated. Harassment of any kind will not be tolerated. Any such remarks will result in an immediate ban.

Reviewer: ```suggestion
1. Be polite, welcoming, and inclusive. 
2. Inappropriate conduct and harassment of any kind will not be tolerated. Any such remarks will result in an immediate ban.
```

I don't think we need to specifically call out the different types of harassment here. It might be worth separating these two into separate points, though.

> 2. Actions that seem specifically designed to test the limits of what is not offensive and serve no other purpose (a.k.a. trolling) will not be tolerated and such sessions will be immediately terminated.

Reviewer: ```suggestion
2. Actions that seem specifically designed to test the limits of what is offensive and serve no other purpose (i.e., trolling) will not be tolerated, and such sessions will be immediately terminated.
```

How do we terminate these sessions?

The Odin Project's online communities are intended to be informative and relaxing spaces where Odinites can find answers to their programming questions or simply hang out.  Since we aim to be an inclusive and friendly community, we request that all Odinites follow the guidelines laid out in this Code of Conduct.  Following these guidelines will ensure that all Odinites have a positive learning experience:

1. Be polite, welcoming, and inclusive. 
2. Inappropriate conduct and harassment of any kind will not be tolerated. Any such remarks will result in an immediate ban.

> 2. Actions that seem specifically designed to test the limits of what is offensive and serve no other purpose (i.e., trolling) will not be tolerated, and such sessions will be immediately terminated.

Reviewer: The numbering of the points gets off here. This one should be `3`, and the next one should be `4`.
